fea-rs/src/parse/lexer.rs

```diff
@@ -446,4 +449,18 @@ mod tests {
         assert_eq!(token_strs[9], "NUM(0)");
         assert_eq!(token_strs[10], "SUFFIX(n)");
     }
+
+    #[test]
+    fn include_with_spaces() {
+        let fea = "include ( path.fea );";
+        let tokens = tokenize(fea);
+        let token_strs = debug_tokens2(&tokens, fea);
+        assert_eq!(token_strs[0], "IncludeKw");
+        assert_eq!(token_strs[1], "WS( )");
+        assert_eq!(token_strs[2], "(");
+        assert_eq!(token_strs[3], "Path( path.fea )");
+        assert_eq!(token_strs[4], ")");
+        assert_eq!(token_strs[5], ";");
+        assert!(token_strs.get(6).is_none());
+    }
```

Comment:
Would love to see the tests here (this and others) change to assert everything at once. A small helper for the tokenenize, debug_tokens2 might be worth considering.

`assert_eq!(vec!["IncludeKw", "WS( )", etc], debug_tokens("fea fragment"));` type of deal.

I see that we sometimes skip indices to avoid writing out the whitespace token, I suggest we either spell it out or add a method to give us the non-whitespace tokens.

Reply:
(I'm find with deferring this to an issue since this PR follows the current convention)

Reply:
ya i think the most elegant solution would be some sort of custom macro...  but this module is touched very rarely, and the tests for actual parsing are generally where this sort of stuff lives, and those have much better ergonomics. 🤷